app/gui/view/graph-editor/src/component/node/expression.rs

```diff
@@ -16,6 +16,8 @@ use span_tree::SpanTree;
 pub struct Expression {
     pub pattern:             Option<String>,
     pub code:                ImString,
+    pub is_skipped:          bool,
+    pub is_frozen:           bool,
```

Comment:
I don't like having those as a part of `Expression`. The fact we're using code for marking a node as skip or freeze is an internal thing I think, not related to the view where this structure is defined.

Reply:
Skipped nodes do not highlight their code, so it is kind of relevant to the view. What is your proposal here?

Reply:
Addressed after a private conversation